PyArc initial plugin (#7)
watts-dev/watts: doc/source/user/usage.rst

        Tsolid_sf = 1e-3
    []

If the templated input file is ``sam_template.inp``, the SAM code will rely the general MOOSE plugin that can be created as::

>     moose_plugin = watts.PluginMOOSE('sam_template.inp', supp_inputs = [])

Reviewer: The default for `supp_inputs` is an empty list, so it doesn't appear that there's any purpose to explicitly specifying that here. I would just list a different example after the next blurb that shows `supp_inputs` being used to specify an additional file.

Author: Agreed. I removed back and added a comment. The option is being shown in another example.

This will generate the OpenMC input files using the template parameters, run
OpenMC, and collect the results.

PyARC Plugin
~~~~~~~~~~~~~

The :class:`~watts.PluginPyARC` class handles PyARC execution in a similar
manner to the :class:`~watts.PluginSAM` class for SAM. PyARC use text-based 

> input files which is templated as shown as follows:

Reviewer: ```suggestion
input files which can be templated as follows:
```

.. code-block:: jinja

    surfaces{
        hexagon ( hex ){ orientation=y   normal = z  pitch = {{ assembly_pitch }} }
        plane ( z0 ) { z = 0.0  }
        plane ( z10 ) { z = {{ assembly_length }} }
    }

> If the templated input file is `pyarc_template`, then PyARC can be executed with the following command:

Reviewer: It's not actually executing PyARC when we create the plugin, so we probably want to choose a different word than "executed"

Author: Agreed. Updated.

>     pyarc_plugin = watts.PluginPyARC('pyarc_template', show_stdout=True, supp_inputs = ['lumped_test5.son'])

Reviewer: ```suggestion
    pyarc_plugin = watts.PluginPyARC('pyarc_template', show_stdout=True, supp_inputs=['lumped_test5.son'])
```
